lib/improver/blending/weighted_blend.py

```diff
@@ -125,17 +238,31 @@ def conform_metadata(
             ndim = cube.coord_dims("time")
             cube.add_aux_coord(forecast_period, data_dims=ndim)
 
+    # update blended cube attributes
+    if "title" not in cube.attributes.keys():
```

Comment:
I'm not sure this should be set here. Either this kind of thing should be coming from the input cubes, or we need to add this in a more centralised way, as we probably want to add it to all IMPROVER cubes from lots of plugins. What do you think?

Reply:
This isn't new, but was moved from `update_blended_cube_metadata` which had already been approved on the basis of the requirements from Caroline's original report.  I agree we really need guidance on metadata!  But until we have it I'm not sure there's any better way to go than approve things that seem reasonably sensible and make sure we document them.